### DISCUSSION TITLE
How do I even enter bios?

### DISCUSSION THREAD

**Discussion by IMTHECOOLEST** [utmapp/UTM#6022] [UTM Help (iOS)]
**Describe the issue**
I'm trying to install windows whistler (beta 2) and it said that I have to enter bios to change the date to continue the installation, can you help?

**Configuration (required)**
* UTM Version: UTM SE V4.4.5
* OS Version: iPadOS 17.0
* Device Model: iPad Pro (11-inch) (3rd generation)
* Is it jailbroken?: No
* How did you install UTM? 
altstore

**Crash log**
If the app crashed, you need a crash log. To get your crash log, open the Settings app and browse to `Privacy -> Analytics & Improvements -> Analytics Data` and find the latest entry for UTM. You should export the text and attach it here.

**Debug log**
For all issues, _including_ crashes, you should attach a debug log. Open UTM, and open the settings for the VM you wish to launch. Near the bottom of the configuration options (or at the top of the `QEMU` page) is `Debug Log`. Turn it on and save the VM. After you experience the issue, quit the VM and re-launch UTM. Open the VM settings again and select `Export Log...` and attach it here.

**Upload VM**
(Optional) If possible, upload the `config.plist` inside the `.utm`. If you do not have this, you can upload the entire `.utm` but note this contains your personal data. Since GitHub has an attachment size limit, you may want to upload to another service such as Google Drive. Link it here.
https://drive.google.com/drive/folders/16knqk5fcBpkjnLQLJosRgWqpyOt80ZgW

**Comment by IMTHECOOLEST** [comment]
The Reason I did the issue is because of time bomb

**Comment by fvvwrd** [comment]
Did you find out how to enter bios?
